### PR TITLE
Connect to api

### PR DESCRIPTION
**Pull request by Christabelle143**
# In this project i completed the following;
- Read the [Bookstore API documentation](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) to learn how to use the API. ✔️ 
- Use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) or [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) for making API calls. ✔️ 
- Fetch book list from Bookstore API on Books page load. ✔️ 
- Add an Action Creator, that returns a function. ✔️ 
- Fetch list of books. ✔️ 
- Dispatch another action (add an action type) that will pass API response to reducer and updates the state. ✔️ 
- Refactor your add book and remove book features to persist your changes in the server. ✔️ 
- Change your existing Action Creators, so they return functions. ✔️ 
- Make changes by using API. ✔️ 
- Dispatch an action (action type that existed before refactor) that will updates the state accordingly. ✔️ 
- Add book and remove book should work in the same way after the refactor. ✔️

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,42 +1,81 @@
...
+    const { title, author, category } = book[0];
+    books.push({
+      id, title, author, category,
+    });
+  });
+  dispatch({
+    type: FETCH_BOOKS,
+    payload: books,
+  });
+  // console.log(books);
```
Comment (Ismailco) [inline src/redux/books/books.js]: - [ ] kindly consider removing all commented code from your repo, it makes your code looks messy and hard to maintain

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Christabelle143, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Christabelle143 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Christabelle143** [comment]
Thanks for review

**Pull request merged**